src/app/pages/world-map/world-map.component.scss

```diff
@@ -0,0 +1,98 @@
+@use "../../../import" as *;
+
+.world_map_container {
+    background-color: #202229;
+    filter: opacity(90%);
+    margin: $globalPaddingTop 0 2rem 0;
+    border-radius: 10px;
+    color: white;
```

Comment:
Use the variable color from -import.scss with variable => color : $textLightColor

Reply:
This folder must not appear in that branch. It's a mistake. I deleted it.